core/rawdb/accessors_indexes.go

```diff
@@ -394,10 +392,8 @@ func DeleteFilterMapLastBlock(db ethdb.KeyValueWriter, mapIndex uint32) {
 	}
 }
 
-func DeleteFilterMapLastBlocks(db ethdb.KeyValueRangeDeleter, maps common.Range[uint32]) {
-	if err := db.DeleteRange(filterMapLastBlockKey(maps.First()), filterMapLastBlockKey(maps.AfterLast())); err != nil {
-		log.Crit("Failed to delete range of filter map last block pointers", "err", err)
-	}
+func DeleteFilterMapLastBlocks(db ethdb.KeyValueStore, maps common.Range[uint32], hashDbSafe bool, stopCallback func() bool) error {
```

Comment:
Can we do the normal deletion rather than using range deleter?

Reply:
In case of map and block pointers of course a normal delete loop would be fine too, actually it does not matter so much performance wise since 99% of the data is the filter rows. But the thing is that for the rows it does make a lot of sense to use `SafeDeleteRange` and once we do that for the rows, it also makes sense to not have any different logic for the pointers. If native `DeleteRange` can be used then it's faster, if the fallback has to be used then the same interrupt/clean up later logic works here too. It's just less extra logic.